src/main/java/edu/cornell/kfs/concur/ConcurParameterConstants.java

```diff
@@ -67,6 +64,5 @@ public class ConcurParameterConstants {
     public static final String CONCUR_EVENT_V2_PROCESSING_REPORT_DETAIL_ITEM = "concur.event.v2.processing.report.detail.item";
     public static final String CONCUR_EVENT_V2_PROCESSING_REPORT_TITLE = "concur.event.v2.processing.report.title";
     public static final String CONCUR_EVENT_V2_PROCESSING_REPORT_FILE_NAME = "concur.event.v2.processing.report.file.name";
-    public static final String CONCUR_EVENT_V2_PROCESSING_REPORT_EMAIL_SUBJECT = "concur.event.v2.processing.report.email.subject";
```

Comment:
Why was this constant and its related resource property removed? They appear to be related to the Event Notification V2 process, not the V1 process. was it because they seemed to be unused?

Reply:
Thanks for the explanation. Given that the constant and its related property appear to be unused, I'm okay with them being removed in this PR.